templates/miq-template-ext-db.yaml

```diff
@@ -265,7 +265,7 @@ objects:
   spec:
     ports:
       -
-        name: "postgresql"
+        name: "postgres"
```

Comment:
I know , I'm inclined over the postgresql naming since that was OpenShift/k8s is doing, after all that's the target platform.